tests/vm_test.go

```diff
@@ -2109,3 +2068,19 @@ func stopVM(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine) *v1.Virtua
 
 	return vm
 }
+
+func patchVmRunStrategy(vm *v1.VirtualMachine, strategy v1.VirtualMachineRunStrategy) error {
+	vm, err := kubevirt.Client().VirtualMachine(vm.Namespace).Get(context.Background(), vm.Name, k8smetav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred())
+
+	oldSpec := vm.Spec
+	newSpec := oldSpec.DeepCopy()
+	newSpec.Running = nil
+	newSpec.RunStrategy = &strategy
+
+	patchData, err := patch.GenerateTestReplacePatch("/spec", oldSpec, newSpec)
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
This function returns an error, please use it instead of asserting.

Reply:
Done here: https://github.com/kubevirt/kubevirt/pull/11948.